pkg/instance/helmfile.go

```diff
@@ -69,13 +71,17 @@ func (h helmfileService) executeHelmfileCommand(accessToken string, instance *mo
 	return cmd, nil
 }
 
-type StackParameters map[string]string
+type stackParameters map[string]string
 
-func (h helmfileService) loadStackParameters(folder string, name string) (StackParameters, error) {
+func (h helmfileService) loadStackParameters(folder string, name string) (stackParameters, error) {
 	environment := h.config.Environment
 	path := fmt.Sprintf("%s/%s/parameters/%s/parameters.yaml", folder, name, environment)
 	data, err := os.ReadFile(path)
 	if err != nil {
+		var e *fs.PathError
```

Comment:
If we want to return empty stack parameters if the file does not exist then

```go
    if errors.Is(err, os.ErrNotExist) {
        return stackParameters{}, err
    }
```

is more accurate. `os.ReadFile()` will return `permission denied` if the file exists but we do not have permission to read it. In that case `errors.As(err, &e)` will also return true.